packages/backend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "backend",
   "scripts": {
-    "test": "jest --passWithNoTests --maxWorkers=50%",
+    "test": "NODE_OPTIONS=--max-old-space-size=4096 jest --passWithNoTests --maxWorkers=50% --logHeapUsage",
```

Comment:
Could we confine these to GitHub Actions? 
https://github.com/interledger/rafiki/blob/d33d4d9b5c3423e7990d0151647062e78942d53b/.github/workflows/lint_test_build.yml#L29
especially `--max-old-space-size`

Reply:
Yeah I was going back and forth on this, but `--max-old-space-size` is pretty device dependent.